src/main/java/duke/Task.java

```diff
@@ -0,0 +1,39 @@
+package duke;
+
+/**
+ * General category
+ */
+public class Task {
+    private String name;
+    private Boolean completed = false;
+
+    public Task(String name) {
+        this.name = name;
+    }
+
+    /**
+     * Mark the task completed
+     */
+    public void doneTask() {
```

Comment:
Setter methods for boolean variables must be of the form:
`void setFound(boolean isFound);`
```suggestion
    public void markTaskAsDone() {
```